test/html/normalize.vows.js

```diff
@@ -13,13 +13,8 @@ wows.describe('html normalizer').addBatch({
     	assert(dom+'', '<div xmlns="http://www.w3.org/1999/xhtml">&lt;123e>&amp;&lt;a<br/></div>');
     	
     	var dom = new DOMParser().parseFromString('<div>&nbsp;&copy;&nbsp&copy</div>','text/html');
```

Comment:
```suggestion
        var dom = new DOMParser().parseFromString('<div>&copy;&nbsp;&copy;</div>>','text/html');
```

(simplify the test string and fix the indentation)

Or do we need to keep the "&nbsp&copy" part to test something? If so, I would kinda favor separate test strings, with separate test assertions, to check the `&nbsp;` behavior and to test with "&nbsp&copy".

I do think it would be better to break this test script into smaller test cases, and potentially multiple files, at some point.

TODO: The assertion would have to be updated as well. Unfortunately the GitHub suggested changes feature does not work for multiple lines if some lines were deleted in the middle.

Reply:
I agree that the tests are not ideal, that's why I raised #72 to discuss the options.

I would avoid to mix this into other PRs to not loose track.